src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -429,17 +429,25 @@ void ProblemManager::parseXMLDocument( xmlWrapper::xmlDocument & xmlDocument )
     for( xmlWrapper::xmlNode regionNode : elementRegionsNode.children() )
     {
       string const regionName = regionNode.attribute( "name" ).value();
-      string const
-      regionMeshBodyName = ElementRegionBase::verifyMeshBodyName( meshBodies,
-                                                                  regionNode.attribute( "meshBody" ).value() );
+      try
+      {
+        string const
+        regionMeshBodyName = ElementRegionBase::verifyMeshBodyName( meshBodies,
+                                                                    regionNode.attribute( "meshBody" ).value() );
 
-      MeshBody & meshBody = domain.getMeshBody( regionMeshBodyName );
-      meshBody.forMeshLevels( [&]( MeshLevel & meshLevel )
+        MeshBody & meshBody = domain.getMeshBody( regionMeshBodyName );
+        meshBody.forMeshLevels( [&]( MeshLevel & meshLevel )
+        {
+          ElementRegionManager & elementManager = meshLevel.getElemManager();
+          Group * newRegion = elementManager.createChild( regionNode.name(), regionName );
+          newRegion->processInputFileRecursive( xmlDocument, xmlDocument, regionNode );
```

Comment:
You provide `xmlDocument` twice?

Reply:
My bad, this mistakes comes from a merge.
Solved.